### PR TITLE
Carthage setup

### PR DESCRIPTION
**Pull request by ayameMBS**
Follow the readme and you should be able to get DroneCore-Swift SDK with Carthage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,49 @@ You should use one of these three ways to import the DroneCore-Swift SDK but not
...
+
+```
+brew install carthage
+```
+
+Then to add the framework, create the file `Cartfile` in your app's repository with below's content:
+
+```
+# Require the iOS framework of DroneCore SDK
+github "dronecore/DroneCore-Swift"
```
Comment (julianoes) [inline README.md]: So this just gets latest master or the latest semver tag?

Comment (julianoes) [reply README.md]: @JonasVautherin you probably need to add a tag here and then create a git tag (or use a branch for now).

Comment (ayameMBS) [reply README.md]: it get the latest version of master I think , but anyway we should add tag

**Comment by ayameMBS** [comment]
@JonasVautherin I have created the tag 0.0.1, do you still get the issue?

**Comment by thomasgubler** [comment]
yay!

**Pull request merged**